Application.js

```diff
@@ -223,13 +233,12 @@ define(["require", "dcl/dcl", "decor/Stateful", "decor/Evented", "dojo/Deferred"
 						dcl.mix(app, window[globalAppName]);
 					}
 					window[globalAppName] = app;
-					app.appStartedDef = appStartedDef;
+					app.appStartedResolve = resolve;
 					app.start();
+				}, function (obj) {
+					reject("Application error back from require for modules message =" + obj.message);
```

Comment:
Does require() really let you specify a callback for errors?   I've never heard of that.